lib/event.js

```diff
@@ -3,96 +3,96 @@
 const util = require('util');
 
 let ao;
-let addon;
+let aob;
 let log;
 
+let eventsCreated = 0;
+let eventsSampled = 0;
+let eventsSent = 0;
+let eventsSendFailed = 0;
+let eventsBytesSent = 0;
+
```

Comment:
Does eventsBytesSent ever get reset? I feel like it could grow very large over time.

Reply:
it can grow to 9007199254740991 before there is an issue. at 1,000,000 bytes/second it can run for 104250 days before it overflows. i think it's ok.